pyspecProcScripts/correlation_alignment.py

```diff
@@ -11,7 +11,9 @@ def to_percent(y, position):
         return s + r'$\%$'
     else:
         return s + '%'
-def correl_align(s, align_phases=False,tol=1e-4,indirect_dim='indirect',fig_title='correlation alignment',ph1_selection=1,ph2_selection=1, sigma = 20,fl=None):
+def correl_align(s, align_phases=False,ODNP=True,tol=1e-4,indirect_dim='indirect',
+        fig_title='correlation alignment',ph1_selection=1,ph2_selection=0, 
```

Comment:
we did have two separate alignment files one for IR which has a 2 x 2 phase cycle and one for the ODNP with a 4 step phase cycle on the ph1 so I made some if/else statements which allows just one script to deal with either shape of the phase cycle by looking at if ph2 is in the dimlabels